src/pages/WhyPage.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+  <div>This is the why page</div>
+</template>
```

Comment:
It's incomplete.
In the issue there is a URL from which you should base yourself to create the Why page.